containers/src/Data/IntMap/Internal.hs

```diff
@@ -1,27 +1,16 @@
-{-# LANGUAGE CPP #-}
-{-# LANGUAGE BangPatterns #-}
-{-# LANGUAGE PatternGuards #-}
-#if __GLASGOW_HASKELL__
-{-# LANGUAGE MagicHash, DeriveDataTypeable, StandaloneDeriving #-}
-{-# LANGUAGE ScopedTypeVariables #-}
-#endif
-#if !defined(TESTING) && defined(__GLASGOW_HASKELL__)
-{-# LANGUAGE Trustworthy #-}
-#endif
-#if __GLASGOW_HASKELL__ >= 708
-{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE CPP, BangPatterns #-}
+#if !defined(TESTING) && __GLASGOW_HASKELL__ >= 703
```

Comment:
```suggestion
#if !defined(TESTING) && defined(__GLASGOW_HASKELL__)
```

The oldest GHC version currently supported is 7.6.

(Not sure what `TESTING` is about BTW)

Reply:
Got it. This is just left over from whatever the CPP guards were when I started this conversion, and I haven't kept track at all of what has changed in regards to compatibility guarantees.